wger/manager/views/workout.py

```diff
@@ -53,6 +57,44 @@
 logger = logging.getLogger(__name__)
 
 
+@login_required
+def export_workouts(request):
+        '''
+        Exports workouts
+        '''
+        # get all workouts
+        workouts = Workout.objects.filter(user=request.user)
+        # generate json data
+        data = serializers.serialize("json", workouts)
+        # create json response
+        response = HttpResponse(data, content_type='text/json')
+        # send response to data.json file
+        response['Content-Disposition'] = 'attachment; filename=data.json'
+        return response
+
+
+def import_workouts(request):
+    file = request.FILES['import_file']
+    data = file.read()
+    f = open('data.json', 'r')
+    import json
```

Comment:
this has already been imported

Reply:
Please elaborate on this, Imported from where?